packages/client/src/subscribe/Subscriber.ts

```diff
@@ -121,7 +121,7 @@ export class Subscriber {
         const subSession = this.getOrCreateSubscriptionSession<T>(streamPartId)
 
         // create subscription
-        const sub = new Subscription<T>(subSession, this.loggerFactory)
+        const sub = new Subscription<T>(subSession.streamPartId, this.loggerFactory)
```

Comment:
Could be simplified.
```suggestion
        const sub = new Subscription<T>(streamPartId, this.loggerFactory)
```